### PR TITLE
Deploy's atomic flag set to false by default

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Deploy's atomic flag set to false by default

### CODE REVIEW

**Review by tobiscr — DISMISSED** [review]

File: docs/gen-docs/kyma_alpha_deploy.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ kyma alpha deploy [flags]
 ## Flags
 
 ```bash
-  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully. (default true)
+  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully.
```
Comment (NHingerl) [inline docs/gen-docs/kyma_alpha_deploy.md:18]:
```suggestion
  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully (default: false).
```

Comment (clebs) [reply docs/gen-docs/kyma_alpha_deploy.md:18]: I can incorporate that to the description, but this is the same thing as happened with Mostafa's PR earlier.
This `(default XXX)` is generated by the tool.

File: cmd/kyma/alpha/deploy/cmd.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ func NewCmd(o *Options) *cobra.Command {
 	}
 
 	cobraCmd.Flags().StringVarP(&o.WorkspacePath, "workspace", "w", defaultWorkspacePath, "Path used to download Kyma sources.")
-	cobraCmd.Flags().BoolVarP(&o.Atomic, "atomic", "a", true, "Use atomic deployment, which rolls back any component that could not be installed successfully.")
+	cobraCmd.Flags().BoolVarP(&o.Atomic, "atomic", "a", false, "Use atomic deployment, which rolls back any component that could not be installed successfully.")
```
[the remark this answers is not in the document]
Comment (clebs) [reply cmd/kyma/alpha/deploy/cmd.go:56]: The issue is that boolean flags can be set several ways to be true.
These forms are equivalent:
- `-a true`
- `-a=true`
- `-a`

Should I document them all?

File: docs/gen-docs/kyma_alpha_deploy.md (hunk 1 above)
Comment (NHingerl) [reply docs/gen-docs/kyma_alpha_deploy.md:18]: then it's fine! Didn't know that

File: cmd/kyma/alpha/deploy/cmd.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (NHingerl) [reply cmd/kyma/alpha/deploy/cmd.go:56]: Let's just document one form, for example `--atomic=true` (or whichever is the most common).

[the remark this answers is not in the document]
Comment (clebs) [reply cmd/kyma/alpha/deploy/cmd.go]: That is the most explicit one, so I put that one on the help text.

**Review by tobiscr — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**